extensions/extensions-helm/aissemble-spark-application-chart/values-migrate.yaml

```diff
@@ -0,0 +1,57 @@
+########################################
+## CONFIG | Spark Configs
+########################################
+metadata:
+  namespace: default
+sparkApp:
+  spec:
+    sparkConf:
+      spark.eventLog.enabled: "true"
+      spark.eventLog.dir: "/opt/spark/spark-events"
+    type: "placeholder" #required for a dry run test to pass, this should always be overridden
+    mode: cluster
+    imagePullPolicy: IfNotPresent
+    restartPolicy:
+      type: Never
+    sparkVersion: "3.4.0"
+    sparkConfigMap: spark-config
+    dynamicAllocation:
+      enabled: true
+      initialExecutors: 0
+      minExecutors: 0
+      maxExecutors: 4
+    volumes:
+      - name: ivy-cache
+        persistentVolumeClaim:
+          claimName: ivy-cache
+      - name: spark-events
+        persistentVolumeClaim:
+          claimName: spark-events-claim
+    driver:
+      cores: 1
+      coreLimit: "1200m"
+      memory: "512m"
+      serviceAccount: spark
+      volumeMounts:
+        - name: ivy-cache
+          mountPath: "/opt/spark/.ivy2"
+        - name: spark-events
+          mountPath: "/opt/spark/spark-events"
+    executor:
+      cores: 1
+      coreLimit: "1200m"
+      memory: "512m"
+      labels:
+        version: 3.4.0
+      volumeMounts:
+        - name: ivy-cache
+          mountPath: "/opt/spark/.ivy2"
+        - name: spark-events
+          mountPath: "/opt/spark/spark-events"
```

Comment:
Q: I see that this is the "alternate" values file that mounts and writes logs to spark history by default, but I guess the spark history events volume just never mentions "history" in its name? Seems to be the case

Reply:
Correct.